native/cocos/platform/android/AndroidPlatform.cpp

```diff
@@ -430,6 +449,18 @@ AndroidPlatform::~AndroidPlatform() = default;
 int AndroidPlatform::init() {
     cc::FileUtilsAndroid::setassetmanager(_app->activity->assetManager);
     _inputProxy = ccnew GameInputProxy(this);
+    _inputProxy->registerAppEventCallback([this](int32_t cmd) {
+        if (APP_CMD_START == cmd || APP_CMD_INIT_WINDOW == cmd) {
+            if (_inputProxy->isAnimating()) {
+                _isLowFrequencyLoopEnabled = false;
+                _loopTimeOut = 0;
+            }
+        } else if(APP_CMD_STOP == cmd) {
```

Comment:
Only enter low frequency mode when stop? When `_inputProxy->isAnimating()` is false, then should enter frequency mode?

Reply:
When the total low frequency duration is exhausted, the game thread will be suspended